docs/external_api/file_manager.md

```diff
@@ -273,10 +272,15 @@ GET /server/files/metadata?filename=tools/drill.gcode
 | `first_layer_extr_temp` |  float   | The configured first layer extruder temperature, in Celsius. |
 | `first_layer_bed_temp`  |  float   | The configured first layer bed temperature, in Celsius.      |
 | `chamber_temp`          |  float   | The configured chamber temperature, in Celsius.              |
-| `filament_name`         |   str    | The name of the filament used.                               |
-| `filament_type`         |   str    | The type of filament used, ie: `PLA`.                        |
+| `filament_name`         |   str    | The name(s) of the filament used.                            |
```

Comment:
For consistency with the documentation it would be preferred for these types to be `string` rather than `str`.   The docs are oriented toward JSON (JavaScript) type descriptions.